ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -132,6 +133,11 @@ protocolInfoByron genesisConfig@Genesis.Config {
                                        $ genesisKeyHashes
               , pbftSignatureThreshold = unSignatureThreshold $
                   fromMaybe defaultPBftSignatureThreshold mSigThresh
+              , pbftSlotLength         = slotLengthFromMillisec
+                                       . fromIntegral
```

Comment:
These conversions can be subtle, but a nice trick to make it clear to the reader is to use an explicit type annotation:
```
  . (fromIntegral :: Natural -> Integer)
```
Then I can see there's no problem here without having to look up what `Update.ppSlotDuration` gives us.

Reply:
I have added the annotation.